### DISCUSSION TITLE
Installing vaex offline

### DISCUSSION THREAD

**Discussion by hjazz6** [vaexio/vaex#1821] [Q&A] [answered]
Hi, 

Is there a way to install vaex offline? I have a server that has no Internet access, and all the installers I've tried tried to connect to the Internet to download additional dependencies.

Thank you.

**Comment by maartenbreddels** [comment]
Hi,

Vaex has additional dependencies, so you would have to download them all. 
Maybe something like https://stackoverflow.com/questions/11091623/how-to-install-packages-offline helps you?

Regards,

Maarten Breddels

**Comment by hjazz6** [comment, marked as answer]
I ended up using `conda install` with the `--download-only` option, downloaded all the packages from an Internet PC, then installed them on my server.
